apps/app/src/views/components/SplitLayout.tsx

```diff
@@ -29,6 +29,7 @@ export const SplitLayout = (props: any) => {
           shadow="md"
           minW="400px"
           alignSelf={isMobileAndTablet ? 'center' : 'flex-start'}
+          mt={2}
```

Comment:
the second child not having any padding or margin causes it to be about 6 pixels higher than the first. i'm adding it here to be equivilant to the first child.